python-threatexchange/threatexchange/exchanges/clients/ncmec/tests/data.py

```diff
@@ -23,6 +23,14 @@
     "&to=2017-10-30T00%3A00%3A00.000Z&start=4001&size=1000&max=5000"
 )
 
+ENTRIES_NO_DATA_XML = """
```

Comment:
Thanks for adding a test!

Reply:
I don't think it's quite what you asked for as we should always get a `GetEntriesResponse` response back or raise an error (meaning it should always enter the for loop)